pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -84,6 +84,20 @@ type LoadBalancerOpts struct {
 	MonitorMaxRetries    uint       `gcfg:"monitor-max-retries"`
 	ManageSecurityGroups bool       `gcfg:"manage-security-groups"`
 	NodeSecurityGroupID  string     `gcfg:"node-security-group"`
+	Provider             string     `gcfg:"provider"`
+
+	// Optional settings that allow a cluster admin to allow his users to set specific LB parameters through Service Annotations
+	EnableAnnotationLBVersion            bool `gcfg:"disable-annotation-lb-version"`
```

Comment:
Oops! Good spot. I've pushed a new commit to make the field tags line up with their respective vars.